proto/panacea/did/v3/tx.proto

```diff
@@ -47,9 +44,7 @@ message MsgUpdateDIDResponse {}
 // MsgDeactivateDID defines the Msg/DeactivateDID request type.
 message MsgDeactivateDID {
   string did = 1;
-  string verification_method_id = 2;
-  bytes signature = 3;
-  string from_address = 4;
+  string from_address = 2;
```

Comment:
> - Add a Deactivated field at DIDDocument and simply replace the related tx.
>
>   - In Deactivation, since there is no document update, it is simply changed to verify only the account signature.

Here, we still have to have a DID authentication (= ownership verification). Without it, anyone can deactivate a certain DID that is not actually owned by him.

You can probably define `MsgDeactivateDID` exactly same as `MsgUpdateDID`, so that you can utilize a `proof` in the `MsgDeactivateDID.document`. If so, a user who tries to deactivate a DID must get his DID document from Panacea first, put a proof into the document, and put the document into the `MsgDeactivateDID`.

Reply:
There is something I misunderstood. As you said, anyone can deactivate it. What do you think is the way to store the panacea account address in `DIDDocument` and compare it when `DeactivateDID`? I understand the method you suggested, but I'm wondering which method is better.

Reply:
We basically try to not couple a DID with a certain chain account, since someone can lost his chain account key while not losing his DID key. That’s why we don’t store a mapping between DID and chain account in Panacea :)

Reply:
Thanks, I understand. I will implement it in the way you suggested.

Reply:
Also, DID has a special concept so called ‘controller’. For example, you can be one of controllers of your kid’s DID, if your DID is written as a controller in your kid’s DID document. Then, you can update/deactivate your kid’s DID document on behalf of your kid. You don’t target this feature yet, but our basic mechanism should be flexible to cover these cases in the future.